code/workspaces/web-app/src/containers/stacks/StacksContainer.js

```diff
@@ -66,13 +66,21 @@ class StacksContainer extends Component {
     // Added .catch to prevent unhandled promise error, when lacking permission to view content
     return this.props.actions.loadStacksByCategory(this.props.projectKey, this.props.containerType)
       .then(() => {
+        // If project key changes in state, another timer seemed to be being created
```

Comment:
Good cach. Yes, line 75 is making a new timeout and just dropping the reference to the old one.